org-roam-bibtex.el

```diff
@@ -447,5 +463,21 @@ notes project before calling any org-roam functions."
             (org-roam-find-file title))
         (message "Something went wrong. Check the *Warnings* buffer.")))))
 
+;;;###autoload
+(defun org-roam-find-non-ref-file (&optional initial-prompt)
+  "Find and open an Org-roam, non-ref file.
```

Comment:
What is "non-ref"?

Reply:
Files which are not reference-notes, i.e. the ones without a `#+ROAM_KEY:`.

Reply:
will be kept for display, right?

Reply:
It's just an extra command to allow people to only show the notes which are not references.  One person in our tracker worried that his references would clutter up his `org-roam-find-file` selection.